dorf_tables.lua

```diff
@@ -0,0 +1,381 @@
+-- dorf_tables has job distribution configurations, random number information for attributes generation, job configurations, profession configurations, dorf type configurations
+-- usage: loaded by pimp-it.lua
+-- by josh cooper(cppcooper) [created: 12-2017 | last edited: 10-2018]
+local help = [====[
```

Comment:
Add: `--@ module = true` somewhere near the top

Reply:
So presumably this is responsible for the variable in the example you linked to:
```lua
if not moduleMode then
    print("scripts/item-descriptions.lua is a content library; calling it does nothing.")
end
```

Reply:
It allows loading the script as a module, which will set moduleMode when it happens. It will also set dfhack_flags.module, which is equivalent.

Reply:
Added it for the next commit